daemon/pod.go

```diff
@@ -62,8 +62,12 @@ func (daemon *Daemon) CmdPodStart(job *engine.Job) error {
 	if _, ok := daemon.PodList[podId]; !ok {
 		return fmt.Errorf("The pod(%s) can not be found, please create it first", podId)
 	}
+	var lazy = false
+	if daemon.Hypervisor == "vbox" && vmId == "" {
```

Comment:
Yes, I created a new pr for runv repo, [https://github.com/hyperhq/runv/pull/3].  These two commits should be integrated together.